sql/schema_info.sql

```diff
@@ -94,3 +94,46 @@ BEGIN
     RETURN time_col_type;
 END
 $BODY$;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.get_chunks(
```

Comment:
Should this function really be in this PR? It doesn't seem to be used anywhere and is SQL so why not leave out?

Reply:
Also, test principle applies here. If this function is not used it should either be taken out or tested. I vote for removing since it is not strictly needed here.

Reply:
removed